src/test/java/jalau/at18/pacman/ConditionMovementsGhostsTest.java

```diff
@@ -0,0 +1,28 @@
+package jalau.at18.pacman;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class ConditionMovementsGhostsTest {
+    // ConditionMovementsGhosts movementG = new ConditionMovementsGhosts();
```

Comment:
Erase the comments if they are not necessary in the future

Reply:
Ok, I'll do it immediately